options/locale/locale_tr-TR.ini

```diff
@@ -1850,6 +1850,13 @@ activity.git_stats_and_deletions=ve
 activity.git_stats_deletion_1=%d silme oldu
 activity.git_stats_deletion_n=%d silme oldu
 
+
```

Comment:
Please dont translate other langs ... it will be overwritten.

This has to be added via crowdin webportal after this pull got merged

Reply:
OK, I won't add any translation from now on. I don't need to delete the existing ones I suppose.

Reply:
Well i would suggest to do so 😅

Reply:
Yeah OK. Since you said it will be overwritten I thought there is no harm if it stays. Anyways, no problem :)